backend/src/bundles/files/file.controller.ts

```diff
@@ -40,16 +47,23 @@ class FileController extends ControllerBase {
 
         this.addRoute({
             path: FileApiPath.UPLOAD,
-            preHandler: uploadFile.single('document'),
+            preHandler: uploadFile.fields([
+                { name: FileGroups.DOCUMENT, maxCount: 1 },
+                { name: FileGroups.IMAGE, maxCount: 1 },
+            ]),
```

Comment:
I think we need just upload endpoint for one document and use it for upload for image and document, because in your current implementation we need to provide document and image

Reply:
I will try to make it more generic.

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/90120572/948d9074-5a62-4222-b0ae-e899a29e72a7)


![Uploading image.png…]()